src/main/java/au/org/ala/biocache/service/DownloadService.java

```diff
@@ -773,6 +776,18 @@ public void writeQueryToStream(DownloadDetailsDTO dd, DownloadRequestParams requ
                     }
                 }
 
+                if (!biocacheDownloadAdditionalLocalFiles.isEmpty()) {
```

Comment:
if `biocacheDownloadAdditionalLocalFiles` is null then a NullPointerException is thrown. 
Needs a null guard.

Reply:
OK, I added a null check, to be more cautious.

Because the spring annotation @Value("${download.additional.local.files:}") cannot be null, it's an empty string by default.

Cheers,
Sylvain